src/authz-module/data/hooks.ts

```diff
@@ -20,17 +21,19 @@ const authzQueryKeys = {
  * React Query hook to fetch all team members for a specific object/scope.
  * It retrieves the full list of members who have access to the given scope.
  *
- * @param object - The unique identifier of the object/scope
+ * @param scope - The unique identifier of the object/scope
+ * @param querySettings - Optional query parameters for filtering, sorting, and pagination
  *
  * @example
  * ```tsx
- * const { data: teamMembers, isLoading, isError } = useTeamMembers('lib:123');
+ * const { data: teamMembers, isLoading, isError } = useTeamMembers('lib:123', querySettings);
  * ```
  */
-export const useTeamMembers = (object: string) => useQuery<TeamMember[], Error>({
-  queryKey: authzQueryKeys.teamMembers(object),
-  queryFn: () => getTeamMembers(object),
+export const useTeamMembers = (scope: string, querySettings: QuerySettings) => useQuery<GetTeamMembersResponse, Error>({
+  queryKey: authzQueryKeys.teamMembers(scope, querySettings),
+  queryFn: () => getTeamMembers(scope, querySettings),
   staleTime: 1000 * 60 * 30, // refetch after 30 minutes
+  refetchOnWindowFocus: false,
```

Comment:
Just curious as to why this is being added. It doesn't seem like something that would be a sort/filter/search specific change.